browser-test/src/end_to_end_enumerators.test.ts

```diff
@@ -1,54 +1,439 @@
-import {test, expect} from '@playwright/test'
+import {test, expect} from './support/civiform_fixtures'
 import {
   AdminPrograms,
   AdminQuestions,
-  createTestContext,
+  enableFeatureFlag,
   loginAsAdmin,
   logout,
   validateAccessibility,
   validateScreenshot,
   waitForPageJsLoad,
 } from './support'
+import {Page} from 'playwright'
 
-test.describe('End to end enumerator test', () => {
+test.describe('End to end enumerator test', {tag: ['@uses-fixtures']}, () => {
   const programName = 'Ete enumerator program'
-  const ctx = createTestContext(/* clearDb= */ false)
 
-  test('Updates enumerator elements in preview', async () => {
-    const {page, adminQuestions} = ctx
-    await loginAsAdmin(page)
+  test.describe('Admin page', () => {
+    test('Updates enumerator elements in preview', async ({
+      page,
+      adminQuestions,
+    }) => {
+      await loginAsAdmin(page)
+
+      await adminQuestions.gotoAdminQuestionsPage()
+
+      await page.click('#create-question-button')
+      await page.click('#create-enumerator-question')
+      await waitForPageJsLoad(page)
+
+      // Click the add button in the preview to ensure we get an entity row and corresponding delete
```

Comment:
**suggestion:** You can use steps for some of these. Probably don't need to migrate the applicant facing non north star ones, but would be great to do it for the admin parts. https://github.com/civiform/civiform/wiki/Playwright-Feature-Migration-Guide#steps

Reply:
done